test/html/expect/test_package+ml/Toplevel_comments/Include_inline/index.html

```diff
@@ -24,11 +24,6 @@ <h1>
   </header>
   <div class="odoc-content">
    <div class="odoc-include">
-    <div class="spec-doc">
-     <p>
-      Doc of <code>T</code>, part 2.
```

Comment:
This is removed because there is a documentation attached but it renders to nothing.
Some elements don't render to anything, for example `@inline` tags.

We could decide it's fine or we could add an exception for for elements that do not render ? cc @dbuenzli 

Unrelated thought: Should we handle tags specifically ?
Currently, it's possible to have tags in the middle of a comment or in the middle of the preamble.
I don't see tags as markup elements like headings and paragraphs but as metadata attached directly to the declarations. I'd expect them on the same hierarchical level as the documentation text, not inside it.

Reply:
> We could decide it's fine or we could add an exception for for elements that do not render ? cc @dbuenzli

I don't understand the question. Also:

> Now, the attached documentation is rendered entirely and the includee top-comment is not.

The top-comment may already have docs, e.g. headings and paragraphs, I think you want these rendered. Did you want to say that you drop the *preamble* ? 

I don't want to sound like a broken record but I think the only way to get to a consistent system is to think about these things in terms of synopse, preamble and content, not in terms of comments.

> I don't see tags as markup elements like headings and paragraphs but as metadata attached directly to the declarations. I'd expect them on the same hierarchical level as the documentation text, not inside it.

That would be my understanding as well. But:

> I don't see tags as markup elements like headings and paragraphs but as metadata attached directly to the declarations.

They are still placed by end-users as they see it fit so I think we should consider them as block-level stuff (and consecutive sequences of tags should define a *single* block, not done yet see #607)

Reply:
On a regular declaration the preamble would be the tag then the paragraph but we agreed in https://github.com/ocaml/odoc/issues/645 that includes are different, there is no preamble ever generated, we either take the decl comment entirely or make a synopsis from T's top-comment.

The problem I'm mentioning is that in this case:

```ocaml
module type T = sig
  (** Top-comment of T. *)
end

include T
(** @inline *)
```

the decl comment renders to nothing but is not technically empty so we won't attempt to take the synopsis out of `T`. Potential fixes:

- Decide it's OK
- Consider tags targeted at Odoc are not part of the doc (they are not rendered currently but are part of the markup: `canonical`, `inline`, `open`, `closed`)
- Handle tags at a different level and render them separately (eg. always after the documentation, wherever they are placed by the user). Like the previous point but applies to every tags and changes the layout.

Reply:
> * Consider tags targeted at Odoc are not part of the doc (they are not rendered currently but are part of the markup: `canonical`, `inline`, `open`, `closed`)

Ah yes thanks for the clarifification. I'd say that this is the right idea.